portal-web/test/functional/com/liferay/portalweb/tests/enduser/staging/usecase/Staging.testcase

```diff
@@ -1189,6 +1189,85 @@ definition {
 		Staging.publishToLive();
 	}
 
+	@description = "This is a use case for LPS-79257 - can publishing single WC / WC Folder to Live."
+	@priority = "3"
+	test PublishWebContentAndFolderIndividually {
+		Staging.openStagingAdmin(siteURLKey = "site-name");
+
+		Staging.activateStagingCP(siteName = "Site Name");
+
+		WebContentNavigator.openWebContentAdmin(siteURLKey = "site-name-staging");
+
+		WebContentNavigator.gotoAddCP();
+
+		WebContent.addCP(
+			webContentContent = "Web Content Content 1",
+			webContentTitle = "Web Content Title 1");
+
+		PortletEntry.publish();
+
+		WebContentFolder.addCP(
+			folderDescription = "WC Folder Description",
+			folderName = "WC Folder Name 1");
+
+		WebContentFolder.addSubfolderCP(
+			folderName = "WC Folder Name 1",
+			subfolderDescription = "WC SubFolder Description",
+			subfolderName = "WC SubFolder Name");
+
+		WebContentNavigator.gotoAddCP();
+
+		WebContent.addCP(
+			webContentContent = "Web Content Content 2",
+			webContentTitle = "Web Content Title 2");
+
+		PortletEntry.publish();
+
+		WebContentNavigator.openWebContentAdmin(siteURLKey = "site-name-staging");
+
+		WebContentFolder.gotoPublish(
+			folderName = "WC Folder Name 1",
+			webContentTitle = "WC Folder Name 1");
+
+		ProductMenu.gotoLiveSite(site = "Site Name");
+
+		WebContentNavigator.openWebContentAdmin(siteURLKey = "site-name");
+
+		AssertElementPresent(
+			key_webContentTitle = "WC Folder Name 1",
+			locator1 = "WC#ENTRY_LIST_TITLE");
+
+		Click(
+			key_webContentTitle = "WC Folder Name 1",
+			locator1 = "WC#ENTRY_LIST_TITLE");
+
+		key_webContentTitle = "Web Content Title 2";
+
+		AssertTextEquals.assertPartialText(
+			locator1 = "WC#ENTRY_LIST_TITLE_ON_LIVE",
+			value1 = "Web Content Title 2");
+
+		AssertElementPresent(
+			key_webContentTitle = "WC SubFolder Name",
+			locator1 = "WC#ENTRY_LIST_TITLE");
+
+		AssertElementNotPresent(
+			key_webContentTitle = "Web Content Title 1",
+			locator1 = "WC#ENTRY_LIST_TITLE_ON_LIVE");
```

Comment:
@chaowang-123 I'll let this go this time, but as a best practice, we shouldn't have functions directly in the testcase file.  If we can wrap them into new macro commands and make them reusable that is best.